### PR TITLE
Update all fornt end files

### PR DESCRIPTION
**Pull request by Gasta-Marc-Oliver**
Hello I just made some revisions to all of my files. Please review my code. Thank you!

### CODE REVIEW

File: gasta_marcoliver/country.js [hunk 1]
```diff
@@ -1,58 +1,34 @@
...
-    regionCountries.innerHTML = `<h3>Countries in the ${region} Region</h3><ul>${countriesHTML}</ul>`;
-  } catch (error) {
-    console.error(error);
-    regionCountries.innerHTML = '<p>Error fetching region countries.</p>';
-  }
-}
\ No newline at end of file
+document.getElementById("search_btn").addEventListener
+  ("click", function () {
+  var country = document.getElementById("country_input").value;
```
Comment (Dupo-JudyAnn) [inline gasta_marcoliver/country.js]: Inconsistent Variable Declaration (Always use "let" keyword in declaring variables over the "var" keyword)

**Review by Dupo-JudyAnn — CHANGES_REQUESTED** [review]

**Review by Gasta-Marc-Oliver — COMMENTED** [review]
Hello I've fixed the code as per suggested. Kindly review it again. Thank you.

**Review by Dupo-JudyAnn — APPROVED** [review]
Hello, I've already checked your code, and I think it's all okay now.

**Comment by Gasta-Marc-Oliver** [comment]
Thank you! @Dupo-JudyAnn

**Pull request merged**